.github/workflows/pull-request.yml

```diff
@@ -76,12 +76,12 @@ jobs:
         run: pnpm cypress install
```

Comment:
If we want to replace PNPM with NPM to increase versatility, should we change all text prompts about pnpm to npm?